js/clinicalTimeline.js

```diff
@@ -797,19 +823,25 @@ var clinicalTimeline = (function(){
   }
 
   timeline.approximateTickToDayValue = function approximateTickToDayValue(tick) {
-    var dayRegex = /(-?\d+)(:?d)/
-    var monthRegex = /(-?\d+)(:?m)/
-    var yearRegex = /(-?\d+)(:?y?)/
-  
-    if (tick.match(dayRegex)) {
-      return parseInt(tick.match(dayRegex)[1])
+    var dayRegex = /(-?)((\d+)(?:y))?((\d+)(?:m))?((\d+)(?:d))/
+    var monthRegex = /(-?)((\d+)(?:y))?((\d+)(?:m))/
+    var yearRegex = /(-?)((\d+)(?:y))/
+
+    var uParse = (num) => num === undefined ? 0 : parseInt(num)
```

Comment:
do we use arrow functions anywhere else in this library? I don't think IE11 supports it: https://caniuse.com/#feat=arrow-functions. Might be better to be safe and avoid it?